jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -162,7 +162,7 @@ def vue_unit_conversion(self, *args, **kwargs):
                 self.app.add_data(converted_spec, new_label)
 
         self.app.add_data_to_viewer(
-            self.app._default_spectrum_viewer_reference_name,
+            self._default_spectrum_viewer_reference_name,
```

Comment:
Sorry about leaving this straggler in there! 

There's one more to catch here: https://github.com/spacetelescope/jdaviz/pull/1839/files#diff-6fd13e1a6ca69516fb610f228b7e5a0f521ea2fbba7682350c3b04c59c8434b7R143

Reply:
Done. Thanks!